_posts/2014-01-30-Repairing-Our-Broken-Specs-Repository.markdown

```diff
@@ -106,6 +108,36 @@ adopters and pod providers. We are very thankful for your patience and
 understanding while we work towards a smoothly functioning version 1 of
 CocoaPods and its architecture.
 
+### I got linked here after updating my gems, what to do?
+
+This issue is being tracked down to an issue located in Psych. To fix it please use the following command:
+
+```shell
+rm -rf "~/Library/Caches/CocoaPods ~/.cocoapods;" \
```

Comment:
@3lvis I was pointing out that the following two are not identical:

```shell
$ echo "hello;" \
   echo "world"
hello; echo world
```

```shell
$ echo "hello"; \
   echo "world"
hello
world
```

You included the semicolon within the quotes so it's passed as an argument to `rm` and not as a command separator.